src/libcore/clone.rs

```diff
@@ -46,14 +46,39 @@
 
 use marker::Sized;
 
-/// A common trait for cloning an object.
+/// A common trait for cloning an object. Differs from `Copy` in that you can
+/// define `Clone` to run arbitrary code, while you are not allowed to override
+/// the implementation of `Copy` that only does a `memcpy`.
```

Comment:
I love your explanation of the distinction between `Clone` and `Copy`, working that in.